### DISCUSSION TITLE
Previewing icon sets

### DISCUSSION THREAD

**Discussion by sbourdeaud** [EvotecIT/PSWriteHTML#198] [Q&A] [answered]
Is there a page where icon sets can be previewed/searched?

**Comment by PrzemyslawKlys** [comment, marked as answer]
For colors I use:
- https://github.com/EvotecIT/PSWriteHTML/blob/ffdca7714aa0a8be8b00fee43aec69f562e940b0/Examples/Example-Colors/Example-Colors.html

For icons I use
- https://github.com/EvotecIT/PSWriteHTML/blob/319c17d04f2c57a4c4ac1d49ec851c53f0aef72c/Examples/Example-Icons/FontAwesome.html

But I do need to create some sort of reference table for easy use
